### DISCUSSION TITLE
Grand Prix Radio Premium

### DISCUSSION THREAD

**Discussion by Marc3l1988** [f1multiviewer/issue-tracker#513] [Feature requests]
Is there a possibility to log in to Grand Prix Radio with a Premium account?

It is currently possible to listen to the Grand radio stream in Multiviewer, but this is just the normal stream from the Netherlands.

Since a couple of months it is possible to get a Premium account to listen to the Belgium stream. It seems that this is available on mobile phones and tablets, but not via a browser.

Can you emulate the app through Multiviewer so we can login to Grand Prix Radio Premium and listen to the right stream?

**Comment by LapsTimeOFF** [comment]
Hey, for now we will only keep using the main Dutch feed, if there is really an interest for us to integrate the Belgian feed, please add a comment in here or contact us in our [Discord](https://muvi.gg/discord)
